macOS/Configuration/App/DuckDuckGoAppStore.xcconfig

```diff
@@ -33,8 +33,15 @@ ENABLE_APP_SANDBOX = YES
 MACOSX_DEPLOYMENT_TARGET = 12.3
 
 PRODUCT_MODULE_NAME = $(APP_STORE_PRODUCT_MODULE_NAME_OVERRIDE:default=$(DEFAULT_PRODUCT_MODULE_NAME))
-PRODUCT_NAME_PREFIX = DuckDuckGo App Store
-PRODUCT_NAME[config=Release][arch=*][sdk=*] = $(RELEASE_PRODUCT_NAME_OVERRIDE:default=$(PRODUCT_NAME_PREFIX))
+PRODUCT_NAME_PREFIX[config=Debug][arch=*][sdk=*] = DuckDuckGo App Store
+PRODUCT_NAME_PREFIX[config=CI][arch=*][sdk=*] = DuckDuckGo
+PRODUCT_NAME_PREFIX[config=Review][arch=*][sdk=*] = DuckDuckGo App Store
+PRODUCT_NAME_PREFIX[config=Release][arch=*][sdk=*] = DuckDuckGo
```

Comment:
I'm not sure this is the exact config we want, so please feel free to question.  This is the actual fix for our translation issues around `InfoPlist.xcstrings` - defining per-build settings is usually handled better by Xcode.